python/libraries/request.py

```diff
@@ -28,4 +28,4 @@ def post(args):
 
 
 def _values():
-	return {"post": {"code": "int", "reason": "str", "text": "str"}, "get": {"code": "int", "reason": "str", "text": "str"}}
+	return {"post": ("str", {"code": "int", "reason": "str", "text": "str"}), "get": ("str", {"code": "int", "reason": "str", "text": "str"})}
```

Comment:
epic, though ideally they'd be async and return `cmd`s (you can mimic discordn.py on the main branch), but I guess I can fix that later

Reply:
Ok ill do that in a bit, but before that we need hashtables or hashmaps, whatever they called